yarn-project/end-to-end/scripts/docker-compose-e2e-sandbox.yml

```diff
@@ -13,8 +13,7 @@ services:
       - '8545:8545'
 
   sandbox:
-    image: $ECR_URL/aztec-sandbox:cache-$COMMIT_HASH
-    #image: aztecprotocol/aztec-sandbox
+    image: aztecprotocol/aztec-sandbox:latest
```

Comment:
Why this change? Shouldn't we be testing an e2e with the sandbox version that corresponds to the current commit, so we ensure that any changes to sandbox are correct before we publish it?

Reply:
Oh, it's because we're tagging the current commit hash locally as `latest` in `run_tests` now, right?

Reply:
Correct